solutions/lesson8.js

```diff
@@ -0,0 +1,13 @@
+const findMax=(obj,max=0)=>{
```

Comment:
create a new parameter for the head. head = linkedL

Reply:
done